3994 reusable page skeleton

GvidasCowi opened the pull request:
- Implemented loading skeleton
- Implemented page skeleton
- Moved TopHeader files to PageSkeleton
- No tests since there aren't any logic in the components

Task [3994](https://dev.azure.com/cowi-business-it/LCA%20-%20Product%20overview/_workitems/edit/3994/)

AlgirdasCOWI commented:
when implementing page skeleton in ProjectAdmin I would expect App to return page skeleton with my passing in 

import React from "react";

const App: React.FC = () => {
  return (
    <PageSkeleton />
  );
};

export default App;

import React from "react";

const PageSkeleton: React.FC = () => {
  return (
    <PageSkeleton>
      <TopHeader/>
      <Sidebar  /> // with CombiunedNavigation component logic inside
      <SubHeader /> with ProjectSelector and UserInfo components inside
      <Content /> //with main content inside that will change based on selected MenuItem from sidebar
    </PageSkeleton>
  );
};

export default PageSkeleton;

Currently you are passing TopHeader in the PageSkeleton. And also,in Sidebar I will pass CombinedNavigation as a single component.

GvidasCowi commented:
In new code commits:
- removed separate components (Sidebar, MainContent)
- PageSkeleton adjusted as in Dialog component to keep same code structure, 
- renamed PageSkeleton to PageLayout to be more self-explanatory

irkscowigithub commented:
hey @GvidasCowi 
4. I find the component confusing, I believe the initial idea was for it to be a meta component that combines other components like _TopHeader_, _ApplicationSpecificMenu_  and _probably Layout/Container_ to create this _PageLayout_ component. 
Something like a shadcn's [combobox ](https://ui.shadcn.com/docs/components/combobox) which combines the _Popover_ and the _Command_ components to derive a new one.
@olisedoc  am I wrong about this ?

@GvidasCowi  It seems that now your component requre to pass on all the structure as a child elements, which kinda defeats the purpose of this component, I think....
With this approach I would expect a quite more extensinve _StoryBook_ documentation demonstrating how to combine _TopHeader_ , _ApplicationSpecificMenu_  etc. to achieve the desired look as seen in the mockups.

GvidasCowi commented:
With new commit:
- Some renaming for the PageLayout components
- adjusted PageLayout stories
- removed unnecessary styling for TopHeader component
- TopHeader moved to original directory;
- PageLayout moved to Layout directory

olisedoc reviewed (dismissed):
👍 looks good, nice that you also adjusted the top header

GvidasCowi commented:
Solved merge conflicts

irkscowigithub commented:
get the latest branch and if tests pass then we will approve it
